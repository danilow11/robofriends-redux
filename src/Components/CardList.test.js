import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import CardList from './CardList';
import React from 'react';

it('expect to render CardList component', () => {
  const mockRobots = [
    {
      id: 1,
      name: 'Dani Alva',
      username: 'danialva',
      email: 'danialva@mail.com'
    }
  ];

  const tree = renderer.create(<CardList robots={mockRobots} />).toJSON();
  expect(tree).toMatchSnapshot();
  // expect(shallow(<Card/>)).toMatchSnapshot();
});
