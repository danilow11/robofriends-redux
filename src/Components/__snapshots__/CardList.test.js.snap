// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `
<div>
  <div
    className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
  >
    <img
      alt="robot"
      src="https://robohash.org/1?200x200"
    />
    <div>
      <h2>
        Dani Alva
      </h2>
      <p>
        danialva@mail.com
      </p>
    </div>
  </div>
</div>
`;
